extensions/ql-vscode/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [UNRELEASED]
 
 - Fix bug when removing databases where sometimes the source folder would not be removed from the workspace or the database files would not be removed from the workspace storage location. [#692](https://github.com/github/vscode-codeql/pull/692)
+- Always pass the `computeDefaultStrings` flag to query compilation commands. This will change the output of some queries. [#694](https://github.com/github/vscode-codeql/pull/694)
```

Comment:
I think I put this into an earlier comment I can't find - I'd prefer to mention what the user-facing change here is, i.e. query results with no string representation were previously omitted, and will now be displayed with placeholder text.

Reply:
Sure. I didn't quite know how to say it. This sounds good.